Flow.Launcher.Plugin/Interfaces/IPublicAPI.cs

```diff
@@ -107,15 +107,15 @@ public interface IPublicAPI
         /// </summary>
         /// <param name="title">Message title</param>
         /// <param name="subTitle">Message subtitle</param>
-        /// <param name="iconPath">Message icon path (relative path to your plugin folder)</param>
+        /// <param name="iconPath">Full path to icon</param>
```

Comment:
Is this comment change correct? Is it relative or full path? I thought it's still some like `./img/icon.png`?

Reply:
It's the full path.

Reply:
relative path also works

Reply:
> relative path also works

Are you sure? iirc I changed this line cuz I found a Python plugin had no icon. maybe it's a Python side issue?

Reply:
https://github.com/Flow-Launcher/Flow.Launcher/blob/026e5f5b0967113ac92c00ad92fa031454834295/Flow.Launcher.Plugin/Result.cs#L63-L81

Maybe the json parser doesn't abide this setter?